tests/NLog.UnitTests/packages.config

```diff
@@ -8,9 +8,11 @@
   <package id="Microsoft.Owin" version="2.0.2" targetFramework="net45" />
   <package id="Microsoft.Owin.Host.HttpListener" version="2.0.2" targetFramework="net45" />
   <package id="Microsoft.Owin.Hosting" version="2.0.2" targetFramework="net45" />
+  <package id="Mono.Data.Sqlite.Portable" version="1.0.3.5" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net45" />
   <package id="Owin" version="1.0" targetFramework="net45" />
   <package id="StatLight" version="1.6.4375" targetFramework="net40" />
+  <package id="System.Data.SQLite.Core" version="1.0.105.2" targetFramework="net40" />
```

Comment:
this should be targetFramework="net35" and  reinstall.

This works for others as those are net2.0 dlls, for example: https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLog.UnitTests.netfx35.csproj#L98